modules/installation-custom-aws-vpc.adoc

```diff
@@ -27,17 +31,37 @@ The installation program no longer creates the following components:
 * VPC DHCP options
 * VPC endpoints
 
-If you use a custom VPC, you must correctly configure it and its subnets for the installation program and the cluster to use. The installation program cannot subdivide network ranges for the cluster to use, set route tables for the subnets, or set VPC options like DHCP, so you must do so before you install the cluster.
+If you use a custom VPC, you must correctly configure it and its subnets for the installation program and the cluster to use. See link:https://docs.aws.amazon.com/vpc/latest/userguide/VPC_wizard.html[Amazon VPC console wizard configurations] and link:https://docs.aws.amazon.com/vpc/latest/userguide/working-with-vpcs.html[Work with VPCs and subnets] in the AWS documentation for more information on creating and managing an AWS VPC.
 
-Your VPC must meet the following characteristics:
+The installation program cannot:
 
-* The VPC's CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address pool for cluster machines.
-* The VPC must not use the `kubernetes.io/cluster/.*: owned` tag.
-* You must enable the `enableDnsSupport` and `enableDnsHostnames` attributes in your VPC so that the cluster can use the Route 53 zones that are attached to the VPC to resolve cluster's internal DNS records. See link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-dns.html#vpc-dns-support[DNS Support in Your VPC] in the AWS documentation. If you prefer using your own Route 53 hosted private zone, you must associate the existing hosted zone with your VPC prior to installing a cluster. You can define your hosted zone using the `platform.aws.hostedZone` field in the `install-config.yaml` file.
+* Subdivide network ranges for the cluster to use.
+* Set route tables for the subnets.
+* Set VPC options like DHCP.
+
+You must do so before you install the cluster. See link:https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Networking.html[VPC networking components] and link:https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html[Route tables for your VPC] for more information on configuring networking in an AWS VPC.
```

Comment:
Optional: 
I got a bit hung up reading, "You must do so before you install the cluster." It make sense and I recognize it's a carry over from the sentence before it was broken up into the bulleted list, but would something along the lines of the following make more sense: 
"You must complete these tasks before you install the cluster."

Reply:
@bmcelvee Good point. It does read a bit awkward. I like your suggestion.